src/logger/SimpleConsoleLogger.ts

```diff
@@ -44,7 +44,7 @@ export class SimpleConsoleLogger implements Logger {
      * Logs query that is slow.
      */
     logQuerySlow(time: number, query: string, parameters?: any[], queryRunner?: QueryRunner) {
-        const sql = query + (parameters && parameters.length ? " -- PARAMETERS: " + this.stringifyParams(parameters) : "");
+        const sql = query + (parameters && (parameters.length > 0 || parameters.length === undefined) ? " -- PARAMETERS: " + this.stringifyParams(parameters) : "");
```

Comment:
can you please explain this `parameters.length === undefined` part?

Reply:
I guess we have it because parameters can be object now? If yes then few concerns: 

* better and more intuitive logic should be applied instead of this code
* do we really need convert mongodb queries into strings in logging?

Reply:
Yes, the mongo parameters are a dictionary indeed, they don't have a length. Did it this way so the minimum amount of code would be changed. You can have own loggers, and I don't know if all loggers would just be fine with having an object in there instead of a string.

Reply:
I agree with @pleerock's think. We should keep it

Reply:
I guess query already string, small problem with parameters.

Reply:
@Matthijn thank you for making it as minimal as possible but we need to improve it a bit to make it clean and maintainable, and improve functionality itself

Reply:
I am new to typescript, but how an object can be assigned to 'any[]'? Shouldn't be only 'any'? Unless it is an array of object so it would have the length defined.